bootstrap.sh

```diff
@@ -5,7 +5,10 @@ set -e
 # be set to "monorepo" in order to the pipeline work properly.
 sub_dir="./"
 if [[ "$PIPELINE_MODE" == "monorepo" ]]; then
-  sub_dir="$REPOSITORY"
+  if [[ -z ${REPOSITORY_PATH} ]]; then
+    $REPOSITORY_PATH = ""
```

Comment:
Para criar variáveis no bash, é sem usar o $ e não pode ter espaço entre o símbolo de =
(sim o bash é chato as vezes kkk)
```suggestion
    REPOSITORY_PATH=""
```

Reply:
hahahaha obrigado pela sugestão, sou leigo em _Bash_